sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -187,6 +188,42 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates a notification when an Application Offering Change Request is completed by the ministry.
+   * @param notification input parameters to generate the notification.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param entityManager optional entity manager to execute in transaction.
+   */
+  async saveApplicationOfferingChangeRequestCompletedByMinistry(
+    notification: ApplicationOfferingChangeRequestCompletedByMinistryNotification,
+    auditUserId: number,
+    entityManager?: EntityManager,
+  ): Promise<void> {
+    const templateId = await this.notificationMessageService.getTemplateId(
+      NotificationMessageType.ApplicationOfferingChangeRequestCompletedByMinistry,
+    );
+    const messagePayload: NotificationEmailMessage = {
+      email_address: notification.toAddress,
+      template_id: templateId,
+      personalisation: {
+        givenNames: notification.givenNames ?? "",
```

Comment:
Not a part of this PR, we can have a common filler method in notification scope as this code is repeating for all methods.